.changeset/olive-sheep-run.md

```diff
@@ -0,0 +1,6 @@
+---
+"@equinor/mad-components": patch
+---
+
+Add useBreakpoint hook to use for apps and components that have specific requirements or styles
```

Comment:
I think changelogs should be written in past tense. For eksempel: Added useBreakpoint hook...

Reply:
not present tense anymore, changed to past tense